examples/circuitplayground_sound_meter.py

```diff
@@ -27,23 +27,32 @@ def normalized_rms(values):
     )
 
 
-mic = audiobusio.PDMIn(
-    board.MICROPHONE_CLOCK, board.MICROPHONE_DATA, sample_rate=16000, bit_depth=16
-)
+# Check to see if the board type is a Circuit Playground Express, and, if so, run the following:
+if cp.circuit_playground_is_type("Express"):
```

Comment:
This will also need another import `from adafruit_circuitplayground.express import Express` and the `cp instantiate will need to be in __init__ (otherwise this import will construct the cpx object.)